osmosis-1/osmosis.zone_assets.json

```diff
@@ -2719,6 +2719,17 @@
       "path": "transfer/channel-20100/nscr",
       "osmosis_verified": false,
       "osmosis_unlisted": true
+    },
+    {
+      "chain_name": "source",
+      "base_denom": "ibc/C97473CD237EBA2F94FDFA6ABA5EC0E22FA140655D73D2A2754F03A347BBA40B",
```

Comment:
This base_denom is wrong. It should be ibc/FC5A7360EEED0713AE3E83E9D55A69AF873056A172AC495890ACE4582FF9685A